<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sign_up_form</class>
 <widget class="QWidget" name="Sign_up_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lblFname">
         <property name="text">
          <string>First name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lnFname"/>
       </item>
       <item>
        <widget class="QLabel" name="lblLname">
         <property name="text">
          <string>Last name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lnLname"/>
       </item>
       <item>
        <widget class="QLabel" name="lblUsername">
         <property name="text">
          <string>User name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lnUsername"/>
       </item>
       <item>
        <widget class="QLabel" name="lblPassword">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lnPassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnSignUp">
           <property name="text">
            <string>Sign up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCancel">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
